pkgs/applications/audio/rsmixer/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, rustPlatform, fetchFromGitHub, libpulseaudio, makeWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rsmixer";
+  version = "0.5.5";
+
+  src = fetchFromGitHub {
+    owner = "jantap";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "HvmX3GCMExkQ74OGiYsDlsNfJk+i258A5seNy3e+wCw=";
+  };
+
+  cargoLock.lockFile = "${src}/Cargo.lock";
+
```

Comment:
```suggestion
```
this should be the default

Reply:
I don't believe this is the case. I tried to build the package without it, however it failed. I can change this to a cargoSha256. I think that the checks are failing because they do not fetch the src, and thus the lockFile is unavailable. That said, I don't see an obvious issue in setting the lockFile to the upstream lock.

Reply:
it might be because it uses [ifd](https://nixos.wiki/wiki/Import_From_Derivation), use `cargoSha256` ore a local copy of `Cargo.lock` if you have to